openedx/core/djangoapps/credit/urls.py

```diff
@@ -19,4 +19,10 @@
         credit_provider_callback,
         name="provider_callback"
     ),
+
+    url(
+        r"^v1/provider/info/(?P<provider_id>[^/]+)/?$",
+        get_provider_detail,
+        name="provider_callback"
```

Comment:
nitpick: please change the name of `url` (`provider_callback`) e.g., `provider_details`